pkg/storage/etcd/etcd_helper.go

```diff
@@ -498,6 +528,12 @@ func (h *etcdHelper) GuaranteedUpdate(
 		if err != nil {
 			return err
 		}
+		newBody := string(data)
+
+		body, err := h.transformer.WriteString(newBody)
```

Comment:
A consistent naming of the transformed and raw value would it easier to read. E.g. body = transformed, value = raw, or conversely. Now it's mixed.